src/test/java/com/iota/iri/benchmarks/crypto/PearlDiverBenchmark.java

```diff
@@ -0,0 +1,34 @@
+package com.iota.iri.benchmarks.crypto;
+
+import com.iota.iri.crypto.PearlDiver;
+import com.iota.iri.utils.Converter;
+import org.openjdk.jmh.annotations.Benchmark;
+
+import java.util.Random;
+import java.util.stream.Collectors;
+
+public class PearlDiverBenchmark {
+  private static final int TRYTE_LENGTH = 2673;
+  private static final int MIN_WEIGHT_MAGNITUDE = 9;
+  private static final int NUM_CORES = -1; // use n-1 cores
+
+  /**
+   * Benchmark Pearl Diver search function.
+   */
+  @Benchmark
+  public void search() {
+    PearlDiver pearlDiver = new PearlDiver();
+    String trytes = getRandomTrytes();
+    byte[] myTrits = Converter.allocateTritsForTrytes(trytes.length());
+    Converter.trits(trytes, myTrits, 0);
+    pearlDiver.search(myTrits, MIN_WEIGHT_MAGNITUDE, NUM_CORES);
+  }
+
+  private String getRandomTrytes() {
+    return new Random()
```

Comment:
I don't really like the idea of randomness in tests (we currently have some randomness in IRI tests but we strive to be rid of that abomination)

You can give `Random` a predetermined seed so it won't be random and change `getRandomTrytes` to `getTrytes`